CryptoPkg/Library/Include/openssl/opensslconf.h

```diff
@@ -17,7 +17,7 @@ extern "C" {
 #endif
 
 /* Autogenerated conditional openssl feature list starts here */
-#if !FixedPcdGetBool (PcdOpensslEcEnabled)

```

Comment:
dumb question: why is this change needed?

Reply:
I'm updating the commit history to make this cleaner but I removed it to ensure that we're always including Elliptic curve cryptography.  It's assumed we always have it in our current implementation.  This is a bit redundant with me setting the PCD to be always TRUE but I thought I might as well be thorough.  I'll add a comment.